src/dbg/commands/cmd-gui.cpp

```diff
@@ -47,7 +48,20 @@ bool cbDebugDump(int argc, char* argv[])
         GuiDumpAtN(addr, int(index));
     }
     else
-        GuiDumpAt(addr);
+    {
+        ACTIVEVIEW activeView;
+        GuiGetActiveView(&activeView);
+        std::string titleString = activeView.title;
+        std::regex dumpPattern("Dump ([1-9][0-9]*)");
+        std::smatch matchResult;
+        if(std::regex_match(titleString, matchResult, dumpPattern))

```

Comment:
Does this really need regex? sscanf should do the trick with `Dump %d`

Reply:
Yeah I’ll fix this once I’m back home.